cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1529,6 +1531,32 @@ function Start_Containerd {
   Log-Output "Starting containerd service"
   Start-Service containerd
 }
+
+$PIGZ_ROOT = 'C:\pigz'
+
+# Install Pigz (https://github.com/madler/pigz) into Windows for improved image
+# extraction performance.
+function Install-Pigz {
+  if ("${env:ENABLE_WINDOWS_PIGZ}" -eq "true") {
+    if (-not (Test-Path $PIGZ_ROOT)) {
+      Log-Output "Installing Pigz for Windows"
+      New-Item -Path $PIGZ_ROOT -ItemType Directory
+      MustDownload-File `
+        -Url 'https://storage.googleapis.com/gke-release/winnode/pigz/prod/gke_windows/pigz/release/5/20201104-134221/pigz-2.3.1.zip' `
+        -OutFile "$PIGZ_ROOT\pigz-2.3.1.zip" `
+        -Hash "5a6f8f5530acc85ea51797f58c1409e5af6b69e55da243ffc608784cf14fec0cd16f74cc61c564d69e1a267750aecfc1e4c53b5219ff5f893b42a7576306f34c" `
+        -Algorithm SHA512
+      Expand-Archive -Path "$PIGZ_ROOT\pigz-2.3.1.zip" `
+        -DestinationPath $PIGZ_ROOT
+      Remove-Item -Path "$PIGZ_ROOT\pigz-2.3.1.zip"
+      Add-MachineEnvironmentPath -Path $PIGZ_ROOT
```

Comment:
Can you add a comment here along the lines of "Putting pigz.exe in the machine's PATH is sufficient for docker to start using it"?

If there's anything more to it than that, please explain. It seems pretty unusual to me that docker behaves this way... I'd think there'd be a config or a flag for this somewhere.